tests/ui/drop_forget_ref.stderr

```diff
@@ -15,6 +15,19 @@ note: argument has type &SomeStruct
 12 |     drop(&SomeStruct);
    |          ^^^^^^^^^^^
 
+warning: call to `std::mem::drop` with a value that implements Copy. Dropping a copy leaves the original intact.
```

Comment:
I've made the drop_copy/forget_copy lints only work if the argument is not a reference type. All 4 lints are tested for in a single pass.